Assets/Scripts/Background/BeyondBorderBackgroundResizer.cs

```diff
@@ -0,0 +1,80 @@
+// Thanks to Loek van den Ouweland for scaling code explanation;
+// their article is where I came across a script with similar code for obtaining camera values and scaling a background.
+// van den Ouweland, L. (2019, October 9). STRETCH A UNITY SPRITE TO FILL THE SCREEN IN A 2D GAME (GAMEOBJECT, NOT UI-CANVAS). 
+// Loek van den Ouweland Software Engineering. Retrieved March 20, 2023, from 
+// https://www.loekvandenouweland.com/content/stretch-unity-sprite-to-fill-the-screen.html.
+// This code increases/decreases the size of the GameObject sprite so that it can cover the empty left/right background space that exists 
+// when the player is using a large/horizontal screen to play the game. The sizing is dependent on the distance between the assigned border's 
+// corresponding side (enum Side) and the same camera border side. NOTE: This script must be applied to 2 filler sprites (Right and Left) to 
+// fill all empty space to the left and right of the game screen. This script was created for tiled continuous sprites in mind, which is why 
+// the size of the sprite is altered, not the scale.
+using UnityEngine;
+
+public class BeyondBorderBackgroundResizer : MonoBehaviour
+{
+    private float sizeFactorWidth;
+    private float sizeFactorHeight;
+    private float borderWidth;
+    private float borderEdge;
+    private float cameraEdge;
+    private float localXScale;
+    private Vector2 currentCameraSize;
+    private SpriteRenderer beyondBorderBgSprite;
+
+    public SpriteRenderer border;  // variable to store border sprite

```

Comment:
I think it's better to use `[SerializeField] private` instead of `public` for variables that are editable in the Inspector. It's easier to analyze the code when variables have more restrictive visibility. For example, you don't have to search for usages of a `private` variable outside of the class that variable is in.

Reply:
Good point, I'll make sure to follow that for future files, as well.